src/Menu.cpp

```diff
@@ -94,6 +94,7 @@ static MenuDef menuDefFile[] = {
     { _TRN("&Open...\tCtrl+O"),             CmdOpen,                   MF_REQ_DISK_ACCESS },
     { "Open Folder",                        CmdOpenFolder,             MF_REQ_DISK_ACCESS | MF_RAMICRO_ONLY },
     { _TRN("&Close\tCtrl+W"),               CmdClose,                  MF_REQ_DISK_ACCESS },
+    { _TRN("&Close All Files"),             CmdCloseAll,               MF_REQ_DISK_ACCESS },

```

Comment:
Consider if could be ALT+F+X  / CTRL + X 
Thus reduce to shorter "E&xit All Tabs\tCtrl+X"),

Reply:
CTRL + X could be accidently triggered while copying some text, I believe that ALT + F + X would be a better choice

Reply:
There is another conflict that needs to be clarified since lower
{ _TRN("E&xit\tCtrl+Q"),                CmdExit, 

Thus X is taken and agree CTRL X is common usage
perhaps  a different combo is needed

Reply:
Maybe some function key + CTRL/ALT? :( This is hard to think

Reply:
I gave a thought about adding a shortcut earlier but realized that it shouldn't be triggered accidently nor should the shortcut be difficult to remember. Sublime Text didn't have a shortcut for closing all files but Notepad++ has CTRL + SHIFT + W

Reply:
So I have added CTRL+SHIFT+W, Should I commit these changes or implement some other shortcut keys?

Reply:
Ctrl W is close windowed tab
 so Ctrl SHIFT close Windowed tabs  sounds good for accelerated exit
just need to decide on the Alt F+ wording

Reply:
Perhaps Alt F T but need to double check
 Close All &Tabs

Reply:
Should Alt F T work the same or the two shortcuts are different?

Reply:
so may need Accelerators.cpp Line after https://github.com/sumatrapdfreader/sumatrapdf/blob/802852a648f05eb5bff360dd3a0d01b03da4e0a0/src/Accelerators.cpp#L28

 {FSHIFT | FCONTROL | FVIRTKEY, 'W', CmdCloseAll},

Reply:
Yes, it's done. I thought you wanted to me add two shortcuts CTRL+SHIFT+W and ALT+F+T

Reply:
ALT F + values are dynamic so &Acrobat &Foxit &Microsoft etc
but can also be different for each language, thus get comments and frequent changes
also there are other examples of non matching equality such as the e&xit = Ctrl Q 
the aim is to minimise "English" confusion by using key words such as Tabs

Reply:
I think I would need your help on this one, I'm making the commit for CTRL+SHIFT+W shortcut for now

Reply:
I dug up my notes for those I use for addins

# Beware many ALT+F+& shortcut combinations are already used thus adding a 2nd
# can add to confusion especially when they may conflict with non-English ones.

# Default Engish Keys = W O C F(x2) S [H] N P A F(x2) I M(MutualyExc.x2) E R X

So I should have known X was already taken

# So MY extras are ALT+F+ M(2) / M(2) / D / L / U / G / Q "/" & #
However **as unofficial** they can be used in Core application

I may have also used some of the following including T but as above these are SumatraPDF+ uses
# Free= B J K T V Y Z  we can also use other basic symbol keys ` = [ ] ; ' \ ,
# Unless on the keypad * - + . Shifted are more dificult ¬!"£$%^&()_{}:@~|<>? 

SO IN SUMMARY it looks like <kbd>ALT</kbd> <kbd>F</kbd> <kbd>T</kbd> is a good candidate

Reply:
Can you suggest the changes to be made and where? lol I really am confused with these shortcuts sorry, but ALT F T is final now